common/server_protocolhandler.cpp

```diff
@@ -373,6 +390,22 @@ void Server_ProtocolHandler::pingClockTimeout()
 
     if (timeRunning - lastDataReceived > server->getMaxPlayerInactivityTime())
         prepareDestroy();
+ 
+    if (QString::fromStdString(userInfo->privlevel()).toLower() == "none")
```

Comment:
This condition check is used in a couple places at this point (during login for example) and now here.  I'm not so sure `userMayBeTimedOutFromInactivity` is the best description. how about something like `userIsUnderPrivileged` ?